app/controllers/api/v1/protected/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V1::Protected::SessionsController < ApplicationController
```

Comment:
Notice the path to this URL endpoint: `api/v1/protected/...`.  I envision some cases where we might expose information via a public endpoint `api/v1/public/...`.

Reply:
Sounds like a good idea to me

Reply:
Might have to refactor this for EmberJS!  I'll test it out soon.